### ISSUE TITLE
Write more descriptive information to the Azure Devops log

### ISSUE THREAD

**Issue by jhoek** [rvanbekkum/ps-xliff-sync#21]
Currently, in Test-XliffTranslation, for each missing and incorrect translation entry, the ID is written to the host (in our case: Azure Devops Pipelines log). 

    $missingTranslationUnits | ForEach-Object {
      Write-Host ($detectedMessage -f $_.id);
    }

This leads to log entries that look like this:

    2021-11-08T14:00:29.3495600Z ##[warning]Missing translation in unit 'Table 476401740 - Field 3249588532 - Property 62802879'.

I think most users would like some influence on the way these things are reported - the actual table ID, field no. and property name would certainly help in the example above. 

My suggestion would be a scriptblock parameter that accepts a single XmlNode (the translation unit) and returns the string that you want to send to Write-Host. The default value for the parameter could be a scriptblock that returns the ID property of the translation unit it receives, so that the out-of-the-box behaviour remains as it is today.

**Comment by rvanbekkum** [comment]
That's a nice idea. I will look into it.

**Comment by jhoek** [comment]
We plan to use this scriptblock for BC translations. Since you module is not exclusively targeted at BC development, I wasn't sure whether to include this as an example.

```
{ param($TranslationUnit) $TranslationUnit.note | Where-Object from -eq 'Xliff Generator' | Select-Object -ExpandProperty '#text' }
```

**Comment by rvanbekkum** [comment]
If it is just as an example (rather than the default value), then that can't hurt, right? 😊

**Comment by jhoek** [comment]
> If it is just as an example (rather than the default value), then that can't hurt, right? 😊

Maybe you could include the example script block in the README?

**Comment by rvanbekkum** [comment]
Surely! 🙂

**Comment by rvanbekkum** [comment]
And perhaps I could look into adding it as the default value for `Test-BcAppXliffTranslations` as that one _is_ targeted at BC development. I think it would be a better default value for BC use-cases. Will most likely take this up as part of #22.

**Comment by rvanbekkum** [comment]
I have just made this the default for the `Test-BcAppXliffTranslations` function with the latest commit in the `develop` branch, i.e., https://github.com/rvanbekkum/ps-xliff-sync/commit/c3d2dbc3f9924b1c7449715a96071d1f291724fa

**Comment by rvanbekkum** [comment]
Shipped in version 1.5.0.0
